File: MaNguonTrenVisualStudio/YNNSHOP56131778/OBJECT/yeu_cau_danh_sach/phan_trang.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication2.YNNSHOP56131778.OBJECT.yeu_cau_danh_sach
{
    public class phan_trang
    {

        public int vi_tri_bat_dau { get; set; }

        public int vi_tri_ket_thuc { get; set; }

        public phan_trang(
        int vi_tri_bat_dau,
        int vi_tri_ket_thuc)
        {

            this.vi_tri_bat_dau = vi_tri_bat_dau;

            this.vi_tri_ket_thuc = vi_tri_ket_thuc;
        }
        public int getvi_tri_bat_dau()
        {
            return vi_tri_bat_dau;

        }
        public int getvi_tri_ket_thuc()
        {
            return vi_tri_ket_thuc;

        }
        public void setvi_tri_bat_dau(int vi_tri_bat_dau)
        {
            this.vi_tri_bat_dau = vi_tri_bat_dau;

        }
        public void setvi_tri_ket_thuc(int vi_tri_ket_thuc)
        {
            this.vi_tri_ket_thuc = vi_tri_ket_thuc;

        }

    }
}

File: MaNguonTrenAndroidStudio/ynn_shop/app/src/main/java/com/ntu/ynn_shop/ynn_shop/cac_doi_tuong/yeu_cau_ds/phan_trang.java
package com.ntu.ynn_shop.ynn_shop.cac_doi_tuong.yeu_cau_ds;

public  class phan_trang{
    int vi_tri_bat_dau;
    int vi_tri_ket_thuc;

    public phan_trang(int vi_tri_bat_dau, int vi_tri_ket_thuc) {
        this.vi_tri_bat_dau = vi_tri_bat_dau;
        this.vi_tri_ket_thuc = vi_tri_ket_thuc;
    }

    public int getVi_tri_bat_dau() {
        return vi_tri_bat_dau;
    }

    public void setVi_tri_bat_dau(int vi_tri_bat_dau) {
        this.vi_tri_bat_dau = vi_tri_bat_dau;
    }

    public int getVi_tri_ket_thuc() {
        return vi_tri_ket_thuc;
    }

    public void setVi_tri_ket_thuc(int vi_tri_ket_thuc) {
        this.vi_tri_ket_thuc = vi_tri_ket_thuc;
    }
}
